src/object_store/src/object/mod.rs

```diff
@@ -140,6 +140,26 @@ pub trait ObjectStore: Send + Sync {
     }
 
     async fn list(&self, prefix: &str) -> ObjectResult<Vec<ObjectMetadata>>;
+
+    fn store_media_type(&self) -> StoreMediaType;
+}
+
+pub enum StoreMediaType {
```

Comment:
This enum looks like a duplicate to ObjectStoreImpl, in which we already differentiate the media type. In addition to using this enum to generate the metric label value, are we planning to use it in other places in the future?

If not, can we just specify a str const in the object store implementation and use it as a label value?

Reply:
Fixed. seems redundant.